mapeditor/mainwindow.cpp

```diff
@@ -1239,3 +1241,36 @@ void MainWindow::on_actionExport_triggered()
 	}
 }
 
+
+void MainWindow::on_actionTranslations_triggered()
+{
+	auto translationsDialog = new Translations(*controller.map(), this);
+	translationsDialog->show();
+}
+
+void MainWindow::on_actionh3m_coverter_triggered()
```

Comment:
```suggestion
void MainWindow::on_actionh3m_converter_triggered()
```
two more times below